src/tests/pages/HomePage.test.js

```diff
@@ -0,0 +1,41 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import { act } from 'react-dom/test-utils';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import Home from '../../pages/Home/Home.page';
+import { store } from '../../store/store';
+
+describe('Testing Home Page', () => {
+  beforeEach(() => {
+    act(() => {
+      render(
+        <Provider store={store}>
+          <MemoryRouter initialEntries={['/home']}>
+            <Home />
+          </MemoryRouter>
+        </Provider>
+      );
+    });
+  });
+  test('Featured Banners is fetching and rendering data from the API', async () => {
+    await waitFor(async () => {
```

Comment:
Please remove the `waitFor` you don't need it, instead you `await screen.find...` selector to wait to finish the render.